src/main/java/ca/serum390/godzilla/config/StartupApplicationConfiguration.java

```diff
@@ -30,13 +34,10 @@
     @EventListener
     public void onApplicationEvent(ApplicationReadyEvent event) {
         log.info("Running post start up configuration...");
-        Flux.just("demo", "jeff", "test")
-                .filterWhen(this::filterPreExistingUsers)
-                .map(this::buildDemoUser)
-                .collectList()
-                .subscribe(users -> godzillaUserRepository.saveAll(users)
-                    .doOnError(log::error)
-                    .subscribe(this::logDemoUserCreation));
+        sendEmailService.sendEmail("amneet.s.270@gmail.com", "test", "this is a test").subscribe();
```

Comment:
Hey can you just reformat this section to how it was before:

```java
        Flux.just("demo", "jeff", "test")
                .filterWhen(this::filterPreExistingUsers)
                .map(this::buildDemoUser)
                .collectList()
                .subscribe(users -> godzillaUserRepository.saveAll(users)
                    .doOnError(log::error)
                    .subscribe(this::logDemoUserCreation));
```

Hard to read with this auto formatting